### ISSUE TITLE
Add unpack tool for ZCS binaries

### ISSUE THREAD

**Issue by halamix2** [namgo/GameWaveFans#3]
* Add decompressor tool for ZCS binaries to unpack (and possibly repack in the future) ZCS files

I can write one and create PR over this week(end)

**Comment by namgo** [comment]
That is awesome! Excellent idea!

**Comment by namgo** [comment]
Just following up from our discussion on the discord. Seems like we've decided to write this in Go.

Reasoning is that golang provides executables and is something we all mostly know. Executables improve accessibility for those interested primarily in unpacking files.

I've worked a little bit with github actions and am happy to integrate that into the tooling for compiliation as needed, no point getting too wrapped up in that yet though.
